### PR TITLE
BREQ-1285-VoithosPreventOverwritingGPOBackups

### PR DESCRIPTION
**Pull request by ruthrosals**
Prevent to create backups if there is an existing backup.

### CODE REVIEW

File: powershell/voithos.psm1 [hunk 1]
```diff
@@ -542,30 +542,37 @@ Function Backup-MountedGPOSettings {
...
+    Write-Host "Backup completed"
+} else {
+    Write-Warning "Can't backup - there is an existing GPO backup on $backupDir"
   }
-  # Unload the hive
-  Write-Host "REG UNLOAD $HKEY_LOCAL_MACHINE_SOFTWARE"
-  REG UNLOAD $HKEY_LOCAL_MACHINE_SOFTWARE
-  Write-Host "Backup completed"
-}
+}
```
Comment (kylep) [inline powershell/voithos.psm1:574]: Wrapping a whole function in an if block only to have an else that does nothing and print an error is a pattern that over time causes the code to get indented really far to the right, and hard to follow.

Instead of
```
function x {
  if thing {
     ...
     ...
     ...
   } else {
     write-warning 'cant do the thing'
     # function returns now
   }
}
```

Try

```
function x {
  if ! thing {
    write-warning 'cant do the thing'
    return
  }
  ...
  ...
  ...
}
```

**Review by kylep — CHANGES_REQUESTED** [review]

**Review by ruthrosals — COMMENTED** [review]
I made some changes can you please verify it? Thanks

**Review by kylep — APPROVED** [review]

**Review by evan-narayan — APPROVED** [review]
